p2ptrans/utils.py

```diff
@@ -1,5 +1,5 @@
 from .config import *
-import math
+from numpy import gcd as np_gcd
```

Comment:
We don't need this anymore if we keep the original gcd. However, we can just get rid on that function and add `np.gcd(int(x),int(y))` directly in the places where it is used. numpy is imported as np basically everywhere in p2p already.

Reply:
Same as before, this should probably be in a different commit/pull resquest since it is unrelated to tests